### DISCUSSION TITLE
How does sshuttle work with the routing table?

### DISCUSSION THREAD

**Discussion by InvisibleFunction** [sshuttle/sshuttle#926] [Q&A]
If I set up my sshuttle connection to forward connections for a specific /32 target, and then I run `netstat -rn -f inet` on my mac, I do not see the target in the route table. Can someone explain how routing decisions are made when sshuttle is in the mix? I've seen some other instances where I set up a target for a /8, and I had hoped more specific routes in my route tables would win out, but they don't seem to.

**Comment by brianmay** [comment]
I think sshuttle operates at the firewall, not the routing table.

Not that familiar with how this works on the Mac. I think it is the same though.
